definitions/infra-harbor/golden_metrics.yml

```
@@ -0,0 +1,28 @@
+httpLiveTraffic:
+  title: "Current (in-flight) HTTP requests"
+  unit: COUNT
+  queries:
+    prometheus:
+      select: latest(harbor_core_http_inflight_requests)
+      from: Metric
```

```suggestion
      from: Metric
      eventId: entity.guid
```


You seem to be missing eventId from our validations:

`[
  {
    "instancePath": "/httpLiveTraffic/queries/prometheus",
    "schemaPath": "#/definitions/query/required",
    "keyword": "required",
    "params": {
      "missingProperty": "eventId"
    },
    "message": "must have required property 'eventId'"
  }
]`

---

This is not needed anymore after having improved the validation code.